manifests/latest/vsphere-7.0u1/vanilla/deploy/validatingwebhook.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: scvalidation-webhook-svc
+  namespace: kube-system
+  labels:
+    app: vsphere-csi-controller
+spec:
+  ports:
+    - port: 443
+      targetPort: 8443
+  selector:
+    app: vsphere-csi-controller
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: scvalidation.csi.vsphere.vmware.com
+webhooks:
+  - name: scvalidation.csi.vsphere.vmware.com
+    clientConfig:
+      service:
+        name: scvalidation-webhook-svc
+        namespace: kube-system
+        path: "/validate-storageclass"
+      caBundle: ${CA_BUNDLE}
+    rules:
+      - apiGroups:   ["storage.k8s.io"]
+        apiVersions: ["v1", "v1beta1"]
+        operations:  ["CREATE", "UPDATE"]
+        resources:   ["storageclasses"]
+    sideEffects: None
+    admissionReviewVersions: ["v1", "v1beta1"]
```

Comment:
Here we're receiving v1beta1 AdminssionReview, but I think the webhook server is only decoding v1 types.

Reply:
we will be configuring webhook on k8s cluster 1.19 or above.

k8s 1.19 already supports `admissionregistration.k8s.io/v1`. I guess we can remove `v1beta1` from the list.
```
% kubectl api-versions | grep admissionregistration
admissionregistration.k8s.io/v1
admissionregistration.k8s.io/v1beta1
```